cns/service/main.go

```diff
@@ -669,6 +663,11 @@ func main() {
 		config.ChannelMode = cns.Managed
 	}
 
+	homeAzMonitor := restserver.NewHomeAzMonitor(nmaClient, time.Duration(cnsconfig.AZRSettings.PopulateHomeAzCacheRetryIntervalSecs)*time.Second)
+	if cnsconfig.ChannelMode == cns.Direct {
```

Comment:
can you add a comment indicating that the homeAz monitor is only required when there is a direct channel between DNC and CNS. This will prevent the monitor from unnecessarily calling NMA APIs for other scenarios such as AKS-swift, swiftv2